src/main/java/com/archisacadeny/course/CourseRepository.java

```diff
@@ -49,6 +49,42 @@ public static Course save(Course course){
         return course;
     }
 
+    public static void createCourseStudentTable() {
```

Comment:
this method should not be here as we discussed in the meeting please do refactor it.